code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -162,6 +166,63 @@ Advanced Voidsuit: Technomancer Exultant
 	desc = "Advanced magnetic boots that have a lighter magnetic pull, placing less burden on the wearer."
 	mag_slow = 1
 
+/***************************************
+Technomancer RIG
+***************************************/
+/obj/item/weapon/rig/techno
+	name = "technomancer RIG suit control module"
+	suit_type = "technomancer RIG suit"
+	desc = "An advanced RIG suit that protects against hazardous, low pressure and high temperature environments."
+	icon_state = "techno_rig"
+	rarity_value = 20
+	armor = list(
+		melee = 40,
+		bullet = 40,
+		energy = 40,
+		bomb = 50,
+		bio = 100,
+		rad = 100
+	)
+	slowdown = 1
+	drain = 3
+	offline_slowdown = 3
+	offline_vision_restriction = 0
+
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+
+	helm_type = /obj/item/clothing/head/space/rig/techno
+	glove_type = /obj/item/clothing/gloves/rig/techno
+	boot_type = /obj/item/clothing/shoes/magboots/rig/techno
+
+	extra_allowed = list(
+		/obj/item/weapon/storage/toolbox,
+		/obj/item/weapon/storage/briefcase/inflatable,
+		/obj/item/device/t_scanner,
+		/obj/item/weapon/rcd
+	)
+
+	req_access = list(access_engine)
```

Comment:
Remove this, it'll cockblock anyone who finds this in the trash.

Reply:
as it should, its techno special item

Reply:
The IH rig isn't locked. The moemed rig isn't locked. The MEO rig isn't locked. Hell, I'm pretty sure even the TE rig isn't locked.

Reply:
Need to lock em then.

Reply:
the meo rig is actually locked

Reply:
My point stands. It's inconsistent and it's better to either lock them all or unlock them all rather than keeping them inconsistent.

Reply:
How do we go about it then?Moemed rig can still be found in trash so I don't wanna lock it, it's bad and not used anyways, I'm not sure about IH one. TE's rig is locked too.

Reply:
Tbh I would recommend removing all access requirements from all RIGs. After that, either lock all factional RIGs out of maint, or allow all of them to spawn in maint with a low chance (aside from possibly the TE's RIG, since that one sounds like it's a unique RIG to me).

Reply:
I'm not adding faction stuff back to maint, that's one thing. The only real faction-specific RIG that's not locker here is IH one, I'd like to hear Gray's/Firefox's/someone else's opinion on that matter.